apps/condo/domains/ticket/utils/clientSchema/Ticket.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Generated by `createschema ticket.Ticket organization:Text; statusReopenedCounter:Integer; statusReason?:Text; status:Relationship:TicketStatus:PROTECT; number?:Integer; client?:Relationship:User:SET_NULL; clientName:Text; clientEmail:Text; clientPhone:Text; operator:Relationship:User:SET_NULL; assignee?:Relationship:User:SET_NULL; classifier:Relationship:TicketClassifier:PROTECT; details:Text; meta?:Json;`
+ */
+
+import { pick, get } from 'lodash'
+
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
+import { generateReactHooks } from '@condo/domains/common/utils/codegeneration/generate.hooks'
+
+import { Ticket as TicketGQL } from '@condo/domains/ticket/gql'
+import { Ticket, TicketUpdateInput, Organization, QueryAllTicketsArgs } from '../../../../schema'
+
+const FIELDS = ['id', 'dv', 'sender', 'deletedAt', 'createdAt', 'updatedAt', 'createdBy', 'updatedBy', 'organization', 'statusReopenedCounter', 'statusReason', 'statusUpdatedAt', 'status', 'number', 'client', 'clientName', 'clientEmail', 'clientPhone', 'unitName', 'watchers', 'operator', 'assignee', 'classifier', 'details', 'related', 'isEmergency', 'isPaid', 'meta', 'source', 'property', 'executor']
+const RELATIONS = ['status', 'client', 'operator', 'assignee', 'classifier', 'organization', 'source', 'property', 'executor', 'related']
+
+interface ITicketUIState extends Ticket {
+    id: string
+    organization: Organization
+}
+
+function convertToUIState (item: Ticket): ITicketUIState {
+    if (item.dv !== 1) throw new Error('unsupported item.dv')
+    return pick(item, FIELDS) as ITicketUIState
+}
+
+interface ITicketUIFormState extends Ticket {
+    organization?: string
+    status?: string
+    source?: string
+    classifier?: string
+    assignee?: string
+    operator?: string
+    client?: string
+}
+
+function convertToUIFormState (state: ITicketUIState): ITicketUIFormState {
+    if (!state) return {} as ITicketUIFormState
```

Comment:
We shouldn't use compile-time casts. Better try to describe more understandable function signature and return type.

Reply:
If you remove this line, there will be problems with auto-completion and the current pages will fall. 
To be honest, you added this line, I tried to remove it, but it breaks the front.